src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/VideoLinkBookingService.kt

```diff
@@ -339,4 +343,27 @@ class VideoLinkBookingService(
       videoLinkBookingEventListener.appointmentUpdatedInNomis(videoLinkAppointment, appointmentChangedEventMessage)
     }
   }
+
+  fun deleteAppointments(releasedOffenderEventMessage: ReleasedOffenderEventMessage) {
+    val reason = releasedOffenderEventMessage.additionalInformation.reason
+    if (reason == Reason.TRANSFERRED || reason == Reason.RELEASED) {
+      val offenderBookings = prisonApiService.getOffenderDetailsFromOffenderNos(
+        listOf(releasedOffenderEventMessage.additionalInformation.nomsNumber),
+        false,
```

Comment:
kotlin style here - would be better to name the parameter as `false` doesn't mean anything to me
```suggestion
        iaActive = false,
```